### PR TITLE
Align image and book viewer

### PR DESCRIPTION
**Pull request by davidpmccormick**
Keeping the journeys consistent between single images and multi-image (e.g. book) works.

### CODE REVIEW

**Comment by davidpmccormick** [comment]
Going to close and wait for @gestchild .

File: catalogue/webapp/pages/item.js [hunk 1]
```diff
@@ -184,7 +211,8 @@ const ItemPage = ({
       {pdfRendering && !mainImageService && (
         <IframePdfViewer title={`PDF: ${title}`} src={pdfRendering['@id']} />
       )}
-      {mainImageService && currentCanvas && navigationCanvases && (
+      {((mainImageService && currentCanvas && navigationCanvases) ||
```
Comment (davidpmccormick) [inline catalogue/webapp/pages/item.js]: is `mainImageService` a necessary check anymore? Doesn't look like we use it inside this conditional.

File: common/views/components/IIIFImagePreview/IIIFImagePreview.js [hunk 2]
```diff
@@ -1,52 +1,69 @@
...
+import NextLink from 'next/link';
+import Image from '../Image/Image';
+import Control from '../Buttons/Control/Control';
 
 type Props = {|
   id: string,
   title: string,
   iiifUrl: string,
   width?: number,
+  itemUrl: any,
```
Comment (davidpmccormick) [inline common/views/components/IIIFImagePreview/IIIFImagePreview.js]: What's the reason for this not being `string`?

Comment (gestchild) [reply common/views/components/IIIFImagePreview/IIIFImagePreview.js]: it's a NextLinkType, have added

File: catalogue/webapp/pages/item.js (hunk 1 above)
Comment (gestchild) [reply catalogue/webapp/pages/item.js]: you're right, will remove

**Pull request merged**